padl/transforms.py

```diff
@@ -36,6 +40,9 @@
 from padl.print_utils import combine_multi_line_strings, create_reverse_arrow, make_bold, \
     make_green, create_arrow, format_argument, visible_len
 
+from IPython.display import Image
+import networkx as nx
```

Comment:
Are we planning to add these as dependencies for padl?

Reply:
For now networkx and ipython is only used for drawing , and further it requires pygraphviz. 
So, will drop this and drawing for now. 
And we can add new feature to draw with minimal requirements in future versions.